assets/js/theme/cart.js

```diff
@@ -90,7 +90,7 @@ export default class Cart extends PageManager {
             invalidEntry = $el.val();
             $el.val(oldQty);
             return swal.fire({
-                text: `${invalidEntry} is not a valid entry`,
+                text: `${invalidEntry} ${this.context.invalidEntryMessage}`,
```

Comment:
**Important**: Could we put string with placeholder into the context and then replace it with invalid entity value?

Reply:
changed.